packages/esm-patient-chart-app/src/visit/visit-form/visit-date-time.component.tsx

```diff
@@ -54,18 +54,18 @@ const VisitDateTimeField: React.FC<VisitDateTimeFieldProps> = ({
                 dateFormat="d/m/Y"
                 datePickerType="single"
                 id={dateFieldName}
-                minDate={minDateObj}
                 maxDate={maxDateObj}
+                minDate={minDateObj}
                 onChange={([date]) => onChange(date)}
                 value={value ? dayjs(value).format('DD/MM/YYYY') : null}
               >
                 <DatePickerInput
                   id={`${dateFieldName}Input`}
+                  invalid={Boolean(errors[dateFieldName])}
```

Comment:
The [invalid](https://github.com/carbon-design-system/carbon/blob/main/packages/react/src/components/DatePicker/DatePicker.tsx#L263) prop expects a boolean.